### DISCUSSION TITLE
Error when uploading proof of my academic status

### DISCUSSION THREAD

**Discussion by yasegor** [community/community#120982] [GitHub Education]
### Hi everyone,

I've been trying to continue my student status on Github for a few days now.
For some reason the student ID verification program rejects my applications. I study in Ukraine, and accordingly my student ID is also filled out in Ukrainian. I translated it through Google Translator and attached the translated document, but I also got rejected. I also combined 2 photos of the original document and the translation into one photo (I saw this recommendation in the community), but I am still getting rejected. For some reason, the program also cannot read the name of my institution, although both the original and the translation match what I indicated earlier in the application.
Can someone contact me to manually confirm my student status?
![Снимок экрана 2024-04-24 152336](https://github.com/community/community/assets/93845348/565b980e-d509-4193-bf89-ad7cf390e357)

**Comment by eliz-amst** [comment]
Does your school name is listed separate? or in a logo only? Is the file you uploaded is a good quality?

**Reply by yasegor** [reply]
Yes, the name of my school is listed separately, there is no logo on my student ID card. And yes, the photo was taken in good quality.

**Reply by eliz-amst** [reply]
Do you have any other form of proof to resubmit? like transcript or proof of enrollment?

**Comment by yasegor** [comment]
@yelyzaveta-samodid No, unfortunately the student card is the only document I have that proves the authenticity of the time until the end of my studies.

**Comment by samus-aran** [comment]
Hi @yasegor. I suggest trying again without the translations and reply when done. We want to confirm the auto-translation is working and then we can see how we can help.

Also take a live capture rather than an upload.
